.github/ISSUE_TEMPLATE/bug-report.yml

```
@@ -0,0 +1,106 @@
+name: Bug Report
+description: Create a bug report
+labels: ["type: bug"]
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for contributing with Quarkus OpenAPI Generator with this bug report! Submit your issue below:
+
+  - type: markdown
+    attributes:
+      value: |
+        ## Bug Report
+
+  - type: dropdown
+    id: extension
+    attributes:
+      label: "Tell us the extension you're using"
+      multiple: false
+      options:
+        - Client
+        - Server
+        - Mock
+    validations:
+      required: true
+
+  - type: input
+    attributes:
+      label: Dependency version
+    validations:
+      required: true
```

```suggestion
  - type: input
    attributes:
      label: Dependency version
    validations:
      required: true
```

Sorry, we can remove this one because in `Environment` we already have it.

---

Lines removed